docs/src/pages/components/snackbars/snackbars.md

```diff
@@ -86,27 +86,20 @@ It also enables you to **stack** them on top of one another (although this is di
 
 (WAI-ARIA: https://www.w3.org/TR/wai-aria-1.1/#alert)
 
-- Since alerts are not required to receive focus, content authors should not require users to close a Snackbar if the role is set to `alert` through the SnackbarContent `role` prop. This is the default role.
-- If a Snackbar requires focus to close it, then content authors should use the `role` of `alertdialog`.
+- By default, the toast won't auto-hide. However, if you decide to use the `autoHideDuration` prop, it's recommended giving users [enough time](https://www.w3.org/TR/UNDERSTANDING-WCAG20/time-limits.html).
```

Comment:
```suggestion
- By default, the Snackbar won't auto-hide. However, if you decide to use the `autoHideDuration` prop, it's recommended giving users [enough time](https://www.w3.org/TR/UNDERSTANDING-WCAG20/time-limits.html).
```

(A toast is an Android system notification)

Reply:
I believe a toast is the generic name for the snackbar (ignoring Material Design language).

Reply:
http://programmerguru.com/android-tutorial/snackbar-vs-toast/
https://stackoverflow.com/questions/34432339/android-snackbar-vs-toast-usage-and-difference

For your SEO kick, you could add *sometimes (mistakenly :wink:) referred to as a "toast"* to the description. But best not to cause confusing by using a different name in the body text.